cli/src/pcluster/config/cluster_config.py

```diff
@@ -1959,6 +1962,8 @@ def __init__(
         self.health_checks = health_checks or HealthChecks(implied=True)
         self.custom_slurm_settings = Resource.init_param(custom_slurm_settings, default={})
         self.tags = tags
+        self.static_node_priority = Resource.init_param(static_node_priority, default=1)
+        self.dynamic_node_priority = Resource.init_param(dynamic_node_priority, default=1000)
```

Comment:
[Nitpick] For coherence we should define these constants in `constants.py` (see DEFAULT_MAX_COUNT a few lines above)

Reply:
I thought about it, and I didn't do it for the following reasons:
- there are several other places where the default is simply written in the parameter definition, without defining a corresponding constant (I agree we should have more consistency on this in the code);
- if a default is used only once in the config and is independent from other default values, I believe defining a constant is pointless and also slightly annoying, since it forces you to follow a redirection to understand what the default value is.

Reply:
ok